package.json

```diff
@@ -68,6 +68,7 @@
     "eslint": "8.42.0",
     "eslint-plugin-node": "11.1.0",
     "jest": "29.7.0",
+    "prettier": "3.2.5",
```

Comment:
Just realized that this is another dependency we will need to maintain. I also often see warnings from them that require constant version updates. 

Should we reconsider?

Reply:
It's just a dev dependency so I would think it's easy to maintain especially with renovate

Reply:
It's the publishing it, then publishing harness-web steps.
Not heavy lifting but adds a distraction to a set of packages going into maintenance.